readme.md

```diff
@@ -1,103 +1,201 @@
 # RSpec
 
 ## Learning Objectives
-- List benefits of unit testing, both to the creation process and to the collaboration process
-- Describe the difference between `context`, `describe`, and `it`
-- Describe the difference between `let`, `before(:each)`, and `before(:all)`
-- Plan the creation of a project by reducing it to a series of unit tests
-- Contrast unit tests and functional tests
-- Add specs to an existing Sinatra app
-- Define Test-Driven Development and give an example of why it's useful
+- Explain the purpose and benefits of Unit testing
+- Describe TDD and it's importance
+- Explain what is RSpec
+- Compare and contrast `expectations` and `matchers`
+- Compare and contrast common RSpec terms including `"describe"`, `"it"`, `"context"`, `before(:each)`, `before(:all)`, `"subject"`,  `"let"`
+- Write and pass unit tests using RSpec
 
-## Why learn this?
+## Framing: Introduction to Test-Driven Development
 
-We've been using the "Sinatra Reloader" gem pretty much since we first touched Sinatra.
+### Why do we implement tests in our applications? (5 min)
 
-#### Why?
+As our applications increase in complexity, we need a safety net.  We need something to ensure that we "Do no harm".  We need a battery of automated tests.  These are specifications about YOUR code that you can run to ensure your code is doing what it should.  
```

Comment:
to increase industry experience, recommend cloning a popular ruby project and running the test suite, like sass - there's like 1000 tests there